code/fmri_utils/anat_preproc/run_anat_preprocess.py

```diff
@@ -0,0 +1,132 @@
+#got most of my guidance for this from the NIPYPE Beginners Guide
+
+import os
+from os.path import dirname, join as pjoin
+import nibabel as nib
+import numpy as np
+from MNI_reorient import MNI_reorient
```

Comment:
and also change the setup.py file to include 'fmri_utils.anat_preproc' as one of the packages